include/crm/msg_xml.h

```diff
@@ -34,6 +34,8 @@ extern "C" {
  */
 
 #define PCMK_XE_DATE_EXPRESSION             "date_expression"
+#define PCMK_XE_REMOTES                     "remotes"
+#define PCMK_XE_REMOTE                      "remote"
```

Comment:
I'm not thrilled about the names or location of these new elements.

Reply:
Maybe reuse `XML_CIB_TAG_NODES` and `XML_CIB_TAG_NODE` (which will eventually get renamed per current style)

Reply:
Yeah, I like those better - at least it's not a new problem.